### PR TITLE
LPS-199447 Search Container list and description views should use Cla…

### PR DESCRIPTION
**Pull request by pat270**
…y checkbox markup

https://liferay.atlassian.net/browse/LPS-199447

### CODE REVIEW

File: portal-web/docroot/html/taglib/ui/search_iterator/list.jsp [hunk 1]
```diff
@@ -246,7 +248,6 @@ if (fixedHeader) {
 
 						textSearchEntry.setAlign(rowChecker.getAlign());
 						textSearchEntry.setColspan(rowChecker.getColspan());
-						textSearchEntry.setCssClass(rowChecker.getCssClass());
```
Comment (ethib137) [inline portal-web/docroot/html/taglib/ui/search_iterator/list.jsp:249]: What was the reason for removing this? Did you see something broken that removing this fixed?

**Review by ethib137 — COMMENTED** [review]
This mostly looks okay to me. It will be better to send this to frontend-infra to review since they own search container and understand it better.

File: portal-web/docroot/html/taglib/ui/search_iterator/list.jsp (hunk 1 above)
Comment (pat270) [reply portal-web/docroot/html/taglib/ui/search_iterator/list.jsp:249]: When I added `rowChecker.setCssClass("custom-control-input");` on line 238, the class name `custom-control-input` gets added to the `td` element and the `input` element. I tried applying `rowChecker.setCssClass("custom-control-input");` after `textSearchEntry`, but the class didn't get applied to the input element 🤷. I'm not sure why.

Comment (ethib137) [reply portal-web/docroot/html/taglib/ui/search_iterator/list.jsp:249]: Okay, @pat270  go ahead and send this to FI for review, explaining that and they should be able to tell you if it's okay or not.

**Comment by ethib137** [comment]
@pat270 can you send this to frontend-infra for them to review?

**Comment by pat270** [comment]
forwarded to https://github.com/liferay-frontend/liferay-portal/pull/3888

**Pull request closed without merging**